trimesh/exchange/threedxml.py

```diff
@@ -144,15 +145,19 @@ def load_3DXML(file_obj, *args, **kwargs):
             (material_file, material_id) = material.attrib['id'].split(
                 'urn:3DXML:')[-1].split('#')
 
-            # triangle strips, sequence of arbitrary length lists
-            # np.fromstring is substantially faster than np.array(i.split())
-            # inside the list comprehension
-            strips = [np.fromstring(i, sep=' ', dtype=np.int64)
-                      for i in faces.attrib['strips'].split(',')]
-
-            # convert strips to (m,3) int
-            mesh_faces.append(util.triangle_strips_to_faces(strips))
-
+            if 'strips' in faces.attrib:
+                # triangle strips, sequence of arbitrary length lists
+                # np.fromstring is substantially faster than np.array(i.split())
+                # inside the list comprehension
+                strips = [np.fromstring(i, sep=' ', dtype=np.int64)
+                          for i in faces.attrib['strips'].split(',')]
+
+                # convert strips to (m,3) int
+                mesh_faces.append(util.triangle_strips_to_faces(strips))
+            elif 'triangles' in faces.attrib:
```

Comment:
The [3D XML User Guide 4.3](http://media.3ds.com/3dxml/3dxmldoc/3D_XML_User_Guide_4.3.pdf) has an example on page 63 with both 'strips' _and_ 'triangles' attributes on the same Face element so `if` may be better than `elif` here.